acados/ocp_nlp/ocp_nlp_common.c

```diff
@@ -122,14 +122,15 @@ ocp_nlp_solver_config *ocp_nlp_solver_config_assign(int N, void *raw_memory)
  * dims
  ************************************************/
 
+// TODO(oj): should be static, but used by current ocp_nlp c++ interface
```

Comment:
all the `calculate_size_self` were there because Robin needed in the interfaces, if we are not going to use them we can clean them out in the future

Reply:
agree!
I guess the breaking changes will come soon enough..